ethereum/core/src/main/java/tech/pegasys/artemis/core/signatures/LocalMessageSignerService.java

```diff
@@ -41,6 +41,11 @@ public LocalMessageSignerService(final BLSKeyPair keypair) {
     return sign(signingRoot);
   }
 
+  @Override
```

Comment:
If all these` sign<insertDataStructure> `methods work on a Bytes object, do we really need multiple methods?

Reply:
Yes, the `RemoteMessageSignerService` sends them to different endpoints.  Ultimately more than just the signing root will likely be provided so that a slashing protection service can be dropped in (eg include the slot for a block).